media-libs/libtiger/libtiger-0.3.4-r1.ebuild

```diff
@@ -27,7 +27,9 @@ src_prepare() {
 
 multilib_src_configure() {
 	ECONF_SOURCE=${S} \
-	econf $(use_enable doc)
+	econf \
```

Comment:
`git grep -C1 ECONF_SOURCE` says otherwise. Most of them either have `econf` on the same line, or unindented on the line following it.